test/e2e/config/vsphere.yaml

```diff
@@ -35,6 +35,8 @@ providers:
         replacements:
           - old: "imagePullPolicy: Always"
             new: "imagePullPolicy: IfNotPresent"
+          - old: "caBundle: Cg=="
```

Comment:
Let's just not forget to drop it with rc.1 again :) (I guess we can already open the PR to bump)